hw/ip/otbn/data/base-insns.yml

```diff
@@ -372,6 +372,8 @@
     The initial value in `grs1` is treated as a bit mask that specifies bits to be set in the CSR.
     Any bit that is high in `grs1` will cause the corresponding bit to be set in the CSR, if that CSR bit is writable.
     Other bits in the CSR are unaffected (though CSRs might have side effects when written).
+
+    If `csr` isn't the index of a valid CSR, this results in an error (setting bit `illegal_insn` in `ERR_BITS`).
```

Comment:
We also need to specify what happens when accessing a CSR without write permissions (which also results in a illegal instruction error).

For CSRRS, the RISC-V specification reads:

> For both CSRRS and CSRRC, if rs1=x0, then the instruction will not write to the CSR at all, and
so shall not cause any of the side effects that might otherwise occur on a CSR write, such as raising
illegal instruction exceptions on accesses to read-only CSRs. 


We should probably take this text almost unmodified.